public/app/features/datasources/components/DataSourceAddButton.tsx

```diff
@@ -1,20 +1,18 @@
-import React from 'react';
+import React, { FC } from 'react';
 
 import { LinkButton } from '@grafana/ui';
 import { contextSrv } from 'app/core/core';
 import { AccessControlAction } from 'app/types';
 
 import { useDataSourcesRoutes } from '../state';
 
-export function DataSourceAddButton() {
+export const DataSourceAddButton: FC = () => {
```

Comment:
Does typescript infer the return without the need for `FC`? I think Grafana prefers to use `JSX.Element | null` over `React.FC` as it adds an implicit type for `children`. 

nit: Do we need to define the [displayName](https://reactjs.org/docs/react-component.html#displayname) as previously it was taken from the function name.

Reply:
I wasn't aware of these things, thanks for pointing them out!